include/zephyr/dt-bindings/gpio/microchip-xec-gpio.h

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright (c) 2022 Microchip Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_MICROCHIP_XEC_GPIO_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_MICROCHIP_XEC_GPIO_H_
+
+/**
+ * @brief Microchip XEC GPIO bank and bit position convenience defines
+ *
+ * Microchip XEC documentation uses octal GPIO pin
+ * numbering. These macros do not require the user to do octal arithmetic
+ * to derive the pin's bit postion.
+ *
+ * Example DT usage: gpios = <MCHP_GPIO176(GPIO_ACTIVE_HIGH)>;
+ *
+ * @{
+ */
+
+#define XEC_GPIO_HELPER(x, y, z)	x y z
+
+/* bank A */
+#define MCHP_GPIO000(x) XEC_GPIO_HELPER(&gpio_000_036, 0, x)
```

Comment:
Something along the lines of:

```suggestion
/* bank A */
#define MCHP_GPIO000 XEC_GPIO_HELPER(&gpio_000_036, 0)
```